src/core/audio/AudioSoundIo.cpp

```diff
@@ -70,19 +70,14 @@ AudioSoundIo::AudioSoundIo( bool & outSuccessful, AudioEngine * _audioEngine ) :
 	const QString& configDeviceId = ConfigManager::inst()->value( "audiosoundio", "out_device_id" );
 	const QString& configDeviceRaw = ConfigManager::inst()->value( "audiosoundio", "out_device_raw" );
 
-	int err;
 	int outDeviceCount = 0;
 	int backendCount = soundio_backend_count(m_soundio);
 	for (int i = 0; i < backendCount; i += 1)
 	{
 		SoundIoBackend backend = soundio_get_backend(m_soundio, i);
 		if (configBackend == soundio_backend_name(backend))
 		{
-			if ((err = soundio_connect_backend(m_soundio, backend)))
-			{
-				// error occurred, leave outDeviceCount 0
-			}
-			else
+			if (int err = soundio_connect_backend(m_soundio, backend); !err)
```

Comment:
Do we need `err` here? I feel like we can just `!soundio_connect_backend(..)` instead?

Reply:
Yeah, we can remove it. I thought it was being used somehow.